### DISCUSSION TITLE
Documentation improvement suggestions

### DISCUSSION THREAD

**Discussion by fvanroie** [HASwitchPlate/openHASP#31] [Q&A]
## 👋 Welcome!
  We’re using Discussions as a place to get feedback from members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nagyrobi** [comment]
I'm sorry if I seem too pushy on some topics.
I deliberately try to approach things from the less experienced user's perspective, trying to rely on existing documentation and thus bring light on missing or errorneous, maybe incomplete parts. Simulating the issues regular users will face when they come in.

If you help me understand right the whole system, I will contribute back to the project by maintaining the documentation up to date, so you can better focus on programming.
